index.bs

```diff
@@ -1557,6 +1556,41 @@ The dictionary SHALL have the following attributes:
 :: The number of sent datagrams that were declared lost, as defined in
    [[!RFC9002]] [Section 6.1](https://www.rfc-editor.org/rfc/rfc9002#section-6.1).
 
+## `WebTransportSendRateStats` Dictionary ##  {#web-transport-sendrate-stats}
+
+The <dfn dictionary>WebTransportSendRateStats</dfn> dictionary includes statistics
+on datagram transmission over the [=underlying connection=].
+
+<pre class="idl">
+dictionary WebTransportSendRateStats {
+  required unsigned long long estimate;
+  required boolean isDataLimited;
+  required boolean isServerLimited;
+  required boolean inSlowStart;
+};
+</pre>
+
+The dictionary SHALL have the following members:
+
+: <dfn for="WebTransportSendRateStats" dict-member>estimate</dfn>
+:: The estimated rate at which queued data will be sent by the user agent, in bits per second.
+   This rate applies to all streams and datagrams that share a [=WebTransport session=]
+   and is calculated by the congestion control algorithm (potentially chosen by
+   {{WebTransport/congestionControl}}).
+: <dfn for="WebTransportSendRateStats" dict-member>isDataLimited</dfn>
+:: True if the rate at which queued data will be sent by the user agent on all streams
+   that share a [=WebTransport session=], as calculated by the
+   congestion control algorithm, is considered application limited. Otherwise false.
+   Datagrams MUST NOT be included in this calculation.
```

Comment:
```suggestion
:: True if the rate at which queued data will be sent by the user agent on all streams
   and datagrams that share a [=WebTransport session=], as calculated by the
   congestion control algorithm, is considered application limited. Otherwise false.
```